.github/workflows/nightly.yml

```diff
@@ -1,9 +1,9 @@
 name: Self-hosted runner with slow tests (scheduled)
 
 on:
-  workflow_dispatch:
-  schedule:
-    - cron: "0 2 * * *"
+  push:
```

Comment:
This is temporary, for GH actions to be run, I need to first push the new workflow file onmain to be able to run it, after my experiments I will revert the commits of this file and keep it untouched